src/v2/tooltip/styles.css.ts

```diff
@@ -0,0 +1,467 @@
+import { withPrefersMotion } from "../../_css-utils";
+
+const TRANSLATE_DISTANCE = vars.spacing.space_1;
+const TRANSLATE_DISTANCE_NEGATIVE = calc.multiply(TRANSLATE_DISTANCE, -1);
+const DURATION = vars.transitionDuration.short;
+const EASING = vars.ease.quart_in_out;
+
+/** -----------------------------------------------------------------------------
+ * OverlayArrow
+ * ------------------------------------------------------------------------------- */
+
+export const overlayArrowCSS = css({ height: "space_3", width: "space_3" });
+
+export const overlayArrowSvgCSS = recipe<
+    ReactAriaRecipe<OverlayArrowRenderProps>
+>({
+    base: [
+        css({
+            display: "block",
+        }),
+        {
+            fill: vars.color.background,
+            stroke: vars.color.border_rule,
+            strokeWidth: 1,
+        },
+    ],
+    variants: {
+        placement: {
+            bottom: {
+                transform: "rotate(180deg)",
+            },
+            center: {},
+            left: {
+                transform: "rotate(-90deg)",
+            },
+            right: {
+                transform: "rotate(90deg)",
+            },
+            top: {},
+        },
+    },
+});
+
+/** -----------------------------------------------------------------------------
+ * Keyframes
+ * ------------------------------------------------------------------------------- */
+
+const origin = createVar();
+
+const keyframesInFromLeft = keyframes({
+    "0%": {
+        opacity: 0,
+        transform: `translateX(${TRANSLATE_DISTANCE_NEGATIVE})`,
+    },
+    "100%": { opacity: 1 },
+});
+
+const keyframesOutToLeft = keyframes({
+    "0%": { opacity: 1 },
+    "100%": {
+        opacity: 0,
+        transform: `translateX(${TRANSLATE_DISTANCE_NEGATIVE})`,
+    },
+});
+
+const keyframesInFromRight = keyframes({
+    "0%": {
+        opacity: 0,
+        transform: `translateX(${TRANSLATE_DISTANCE})`,
+    },
+    "100%": { opacity: 1 },
+});
+
+const keyframesOutToRight = keyframes({
+    "0%": { opacity: 1 },
+    "100%": {
+        opacity: 0,
+        transform: `translateX(${TRANSLATE_DISTANCE})`,
+    },
+});
+
+const keyframesInFromTop = keyframes({
+    "0%": {
+        opacity: 0,
+        transform: `translateY(${TRANSLATE_DISTANCE_NEGATIVE})`,
+    },
+    "100%": { opacity: 1 },
+});
+
+const keyframesOutToTop = keyframes({
+    "0%": { opacity: 1 },
+    "100%": {
+        opacity: 0,
+        transform: `translateY(${TRANSLATE_DISTANCE_NEGATIVE})`,
+    },
+});
+
+const keyframesInFromBottom = keyframes({
+    "0%": {
+        opacity: 0,
+        transform: `translateY(${TRANSLATE_DISTANCE})`,
+    },
+    "100%": { opacity: 1 },
+});
+
+const keyframesOutToBottom = keyframes({
+    "0%": { opacity: 1 },
+    "100%": {
+        opacity: 0,
+        transform: `translateY(${TRANSLATE_DISTANCE})`,
+    },
+});
+
+export const popoverCSS = recipe<ReactAriaRecipe<TooltipRenderProps>>({
```

Comment:
Is this actually being used? Looks like it isn't? 

(Aside: I think adding `knip` as a check to this repo would prob be a good shout)